prime-dotnet-webapi/Services/HealthAuthorityService.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Prime.Models;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using AutoMapper;
+using AutoMapper.QueryableExtensions;
+
+using Prime.ViewModels.Parties;
+
+namespace Prime.Services
+{
+    public class HealthAuthorityService : BaseService, IHealthAuthorityService
+    {
+        private readonly IMapper _mapper;
+        public HealthAuthorityService(
+            ApiDbContext context,
+            IMapper mapper,
+            IHttpContextAccessor httpContext)
+            : base(context, httpContext)
+        {
+            _mapper = mapper;
+        }
+
+        public async Task<IEnumerable<AuthorizedUserViewModel>> GetAuthorizedUsersByHealthAuthorityAsync(HealthAuthorityCode code)
+        {
+            return await _context.AuthorizedUsers
+                .Where(u => u.HealthAuthorityCode == code)
+                .ProjectTo<AuthorizedUserViewModel>(_mapper.ConfigurationProvider)
```

Comment:
Going by whats in the FE:
![image](https://user-images.githubusercontent.com/39168456/118105723-9cf71380-b391-11eb-9efe-75f621753bfe.png)

This viewmodel only needs Name, Job title, Id and Status. I assume this method is re-using the AuthorizedUserViewModel to keep it really small because we expect that we are removing this controller and service in the near future? If that's the case, could we add a comment to that affect somewhere to avoid confusion? (Also, this wont populate the Addresses because they are added in Includes() rather than the mapping configuration)